openpype/hosts/nuke/api/plugin.py

```diff
@@ -669,7 +670,7 @@ def __init__(self,
         self.ext = ext or "cube"
         self.cube_size = cube_size or 32
         self.lut_size = lut_size or 1024
-        self.lut_style = lut_style or "linear"
+        self.lut_style = lut_style or "scene_linear"
```

Comment:
I changed this a bit ago (also in some other places where `linear` is hard-coded) as newer OCIO configs don't have `linear` as apparently it was decided that `linear` was too vague and confusing. I can remove the change as it's out of scope but since I commited it (by mistake) I might as well raise the discussion